### DISCUSSION TITLE
Downlod Complete TIC-80 Library for Offline Use and Archival

### DISCUSSION THREAD

**Discussion by LinuxinaBit** [nesbox/TIC-80#2580] [General]
Would it be possible to somehow download the entire TIC-80 library on https://tic80.com/play into a single folder for offline use and archival, and have it be accessible from inside TIC-80 in a similar way to `surf`ing the website when online (possibly with program thumbnails as well)?

**Comment by Skeptim** [comment]
Note that in version 1.2 the 'autosave' option in the menu allows to save automatically games when loading them. Then you can access them using `surf` offline.
However it won't download the entire library.

But someone did that two years ago by scrapping the website https://www.reddit.com/r/Roms/comments/yzllvp/tic80_all_3039_carts_scraped_with_cover_art/ you could contact them to get the tools.

**Comment by bbroke** [comment]
With some adjustments you could use this script https://gist.github.com/bbroke/38a55953aef117c8e7e5956d743ad95c
